src/tools/bin/nextstep_from_period.sh

```diff
@@ -19,22 +19,52 @@
 # If not, see <http://www.gnu.org/licenses/>.
 #
 
-# Script returns next timestep specified by the period syntax
-#
-# Takes three lines with values from stdin:
-# - a string representing period-syntax, like `3000,12000:15000:1000,1337:1337`
-# - the current timestep (convertible to int)
-# - and the final timestep (convertible to int)
-#
-# and prints the next timestep to stdout
+set -euf -o pipefail
+
+helpText()
+{
+    echo "
+    Usage: nextstep_from period period lastStep [currentStep]
 
-read period
-read currentStep
-read lastStep
+    Script returns next timestep specified by the period syntax
+    Takes three positional arguments:
+     - a string representing period-syntax, like 3000,12000:15000:1000,1337:1337
+     - the current timestep (convertible to int)
+     - and the final timestep (convertible to int)
 
-if [ -z "$currentStep" ]; then
+    and prints the next timestep to stdout"
+}
+
+case $# in
+  0)
+    printf "\n *** Not enough arguments! *** \n"
+    helpText
+    exit 1
+  ;;
+  1)
+    helpText
+    if [ "$1" = "-h" ] || [ "$1" = "-h" ]; then
```

Comment:
question:
what is the difference between both or-cases?
Should it be instead as follows?
```bash
if [ "$1" = "-h" ] || [ "$1" = "--help" ];
```